recipes/check_shapes/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "check_shapes" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b699fcb1e60bb17e2c97007e444b89eeeea2a9102ff11d61fb52454af5348403
+
+build:
+  number: 0
+
+outputs:
+  - name: check_shapes
+    build:
+      noarch: python
+      script: python -m pip install . -vv
```

Forgot that there is a conda-build bug ( https://github.com/conda/conda-build/issues/3993 ) where `pip` is not configured properly to disable downloads during the package build and the like. That bug was recently fixed, but conda-build hasn't released yet. For now, we need to configure `pip` ourselves. Would suggest something like this, which just does what conda-build will do (once it is fixed). After the next conda-build release, this can be dropped

```suggestion
      script_env:
        # Workaround an upstream conda-build issue w/pip & `outputs` by setting env vars manually.
        # xref: https://github.com/conda/conda-build/issues/3993
        - PIP_NO_BUILD_ISOLATION=False
        - PIP_NO_DEPENDENCIES=True
        - PIP_IGNORE_INSTALLED=True
        - PIP_CACHE_DIR=pip_cache
        - PIP_NO_INDEX=True
      script: python -m pip install . -vv
```